oss_src/unity/lib/unity_sarray.cpp

```diff
@@ -705,30 +705,36 @@ size_t unity_sarray::num_missing() {
 
 bool unity_sarray::all() {
   log_func_entry();
-  
+  class early_termination{};
```

Comment:
To make clearer what's going on, maybe this should be early_termination_return_zero?